shanoir-ng-front/src/app/shared/mass-download/download-setup-alt/download-setup-alt.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="cell">
+    <form #window class="window" [formGroup]="form" *ngIf="form">
+        <h2 class="header">Download Datasets</h2>
+        <div class="left-icon msg">
+            <i class="fa-solid fa-circle-exclamation"></i>
+            <b>Limited functionalities : </b>
```

Comment:
@michaelkain, I don't remember what we said around this ?  Maybe not in red ?
![image](https://github.com/fli-iam/shanoir-ng/assets/7604176/86fd378e-f7bf-4614-8a12-39911ac7697b)